src/Microsoft.Health.Fhir.Shared.Api/Controllers/ExportController.cs

```diff
@@ -229,8 +264,13 @@ private void CheckIfAnonymizedExportIsEnabled()
             }
         }
 
-        private static void CheckContainerNameForAnonymizedExport(string containerName)
+        private static void CheckContainerNameAndConfigLocationForAnonymizedExport(string containerName, string anonymizationConfigLocation)
         {
+            if (string.IsNullOrWhiteSpace(anonymizationConfigLocation))
```

Comment:
nit: Would it be helpful to add UTs for this part of the code? Or are they covered by E2E tests?

Reply:
Add unit tests for there 2 Exception in ExportControllerTests.